draft-ietf-quic-http.md

```diff
@@ -556,7 +558,8 @@ extension could negotiate a modification of this restriction; see
 Pseudo-header fields are only valid in the context in which they are defined.
 Pseudo-header fields defined for requests MUST NOT appear in responses;
 pseudo-header fields defined for responses MUST NOT appear in requests.
-Pseudo-header fields MUST NOT appear in trailers.  Endpoints MUST treat a
+Pseudo-header fields MUST NOT appear in trailer sections.
```

Comment:
IIUC the "trailers" vernacular refers to trailer fields, not trailer sections.  The preceding change was made since I was already looking at "trailers" usage, though the existing text doesn't seem incorrect in its own right.